packaging/greenboot/microshift-running-check.sh

```diff
@@ -28,7 +28,43 @@ function return_failure() {
 # args: None
 # return: None
 function script_exit() {
-    [ "$?" -ne 0 ] && echo "FAILURE" || echo "FINISHED"
+    if [ "$?" -ne 0 ] ; then
+        # Always log the last 1000 MicroShift and etcd journal entries
```

Comment:
Why do we want so much output?

I think we'll serve the user better by giving them a concise message and helping them understand where to go to debug next. Dumping all of the log output from MicroShift into the health check log is going to make it harder to debug failures unrelated to MicroShift.

Reply:
Agreed, I overdid it - removing.